inspirehep/modules/theme/views.py

```diff
@@ -101,3 +117,35 @@ def data():
 @blueprint.route('/ping')
 def ping():
     return 'OK'
+
+
+@blueprint.route('/postfeedback', methods=['POST', ])
+def postfeedback():
+    """Handler to create a ticket from user feedback."""
+    feedback = request.form.get('feedback')
+    if feedback == '':
+        return make_response(jsonify(success=False), 400)
+
+    replytoaddr = request.form.get('replytoaddr', None)
+    if replytoaddr is None:
+        if current_user.is_anonymous:
+            return make_response(jsonify(success=False), 403)
+        else:
+            replytoaddr = current_user.get('email')
+            if replytoaddr == '':
+                return make_response(jsonify(success=False), 403)
+
+    content = '''Feedback:
+{feedback}'''.format(feedback=feedback)
+    message = {
+        'sender': current_app.config['CFG_SITE_SUPPORT_EMAIL'],
```

Comment:
Minor thing: these variables are missing in the current `inspirehep/config.py`, so errors will be thrown out of the box. However, I dunno if we shall add default values to `config.py` or leave it for people to configure themselves when they see the error.